package.json

```diff
@@ -41,7 +41,7 @@
     "typescript": "^4.7.4"
   },
   "dependencies": {
-    "@notionhq/client": "^2.0.0",
+    "@notionhq/client": "^2.2.14",
```

Comment:
had to upgrade to get better TS support.
I already use this in another project and no breaking change so far.

Reply:
makes sense in any case, thanks 👍